plugins/output-csv/pom.xml

```diff
@@ -32,7 +32,7 @@
         <dependency>
             <groupId>com.databricks</groupId>
             <artifactId>spark-csv_2.11</artifactId>
-            <version>1.0.3</version>
+            <version>1.1.0</version>
```

Comment:
This version is for Spark 1.4. Please use other (I think v1.0.3 is the correct one)